drivers/SmartThings/matter-lock/fingerprints.yml

```diff
@@ -10,9 +10,15 @@ matterManufacturer:
     vendorId: 0x135d
     productId: 0xb0
     deviceProfileName: lock-nocodes-notamper
+#Aqara
+  - id: "4447/10242"
+    deviceLabel: Aqara Smart Lock U200
+    vendorId: 0x115F
+    productId: 0x2802
+    deviceProfileName: lock-nocodes-nobattery-lockAlarm
 matterGeneric:
   - id: "matter/door-lock"
     deviceLabel: Matter Door Lock
     deviceTypes:
       - id: 0x000A # Door Lock
-    deviceProfileName: base-lock
+    deviceProfileName: base-lock #TODO update if lockAlarm is visible
```

Comment:
I think we want to match the new Aqara profile whatever we do end up using there, except probably with battery since most locks are going to be battery powered

Reply:
Then can we apply the lock-lockalarm profile here?

Reply:
[d0a77c9](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1413/commits/d0a77c936e3ea9e7dfbf99220f3f0fa63051754b) updated this to use the `lock-lockalarm` capability, which is the same as the `lock-lockalarm-nobattery` used the the aqara lock but it has battery support